src/plone/restapi/tests/test_atcontent_serializer.py

```diff
@@ -193,3 +194,119 @@ def test_get_is_folderish_in_folder(self):
         obj = serializer()
         self.assertIn("is_folderish", obj)
         self.assertEqual(True, obj["is_folderish"])
+
+    def test_nextprev_alone(self):
+        folder = api.content.create(
+            container=self.portal,
+            type="Folder",
+            title="Folder with items",
+            description="This is a folder with some documents"
+        )
+        doc = api.content.create(
+            container=folder,
+            type="Document",
+            title="Item 1",
+            description="One item alone in the folder"
+        )
+        data = self.serialize(doc)
+        self.assertEqual(data["previous_item"], None)
```

Comment:
None does not seem right. An empty array should be more suited.

Reply:
assertEqual should take the expected output first, and the calculated input second.

Reply:
you mean empty dict, right?